### ISSUE TITLE
Randomness in validation loss?

### ISSUE THREAD

**Issue by tbjorkl5** [nasa/pretrained-microscopy-models#13]
I see a worryingly high fluctuation in validation loss that never seem to stabilize. Even if I set LR = 1e-99 in multiclass_segmentation_materials.ipynb in the finetuning step it's varying between 0.95 down to 0.5 between epochs, why is there any variation at all?

**Comment by JStuckner** [comment]
That is strange. Some fluctuation should be expected, but I wouldn't expect much fluctuation at such a low learning rate. Is it possible you're applying image augmentation to the validation set or somehow making any changes to what exact images are being used for validation between epochs? Or maybe applying dropout on validation?
